precompile/living-assets/src/tests.rs

```diff
@@ -81,7 +121,7 @@ fn create_collection_assign_collection_to_caller() {
 #[test]
 fn owner_of_with_nonzero_transfer_should_fail() {
```

Comment:
This test must be removed now, after removing owner_of.

Reply:
right!